WiseUnpacker/SabreTools.Serialization.Deserializers/WiseSectionHeader.cs

```diff
@@ -78,6 +76,13 @@ public class WiseSectionHeader : BaseBinaryDeserializer<SectionHeader>
 
                 data.Seek(initialOffset + offset - versionOffset, 0);
                 header.Version = data.ReadBytes(versionOffset);
+                localWisOffset = offset;
+            }
+            
+            if (header.Version == null)
+            {
+                // TODO: error output
```

Comment:
For better or worse, parsing methods don't tend to have console writes when something goes wrong. You can just have it return null.

Reply:
That's fair. I had kind of left it there as a reminder of something else that needs to be done for all the return null's, but it should indeed be removed.

Reply:
Should be fixed.